src/src/hooks/Teams.js

```diff
@@ -0,0 +1,46 @@
+import { useSelfServiceRequest } from "./SelfServiceApi";
+import { useEffect, useState } from "react";
+
+export function useTeams() {
+  const { responseData: loadResponse, sendRequest: loadTeams } =
+    useSelfServiceRequest();
+  const { responseData: addResponse, sendRequest: addTeam } =
+    useSelfServiceRequest();
+  const [teams, setTeams] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    loadTeams({
+      urlSegments: ["teams"],
```

Comment:
This is not a blocker, but it is something I realised when writing the invitations:

This approach actually counter-acts our _links approach.
I am sorry I did not see that before, but for now let's just continue.